app/views/registration_wizard/start.html.erb

```diff
@@ -8,7 +8,7 @@
 
     <h2 class="govuk-heading-m">Before you start</h2>
 
-    <p class="govuk-body"><%= govuk_link_to("Choose an NPQ and provider", choose_an_npq_and_provider_path) %>.</p>
+    <p class="govuk-body"><a href="https://professional-development-for-teachers-leaders.education.gov.uk/" class="govuk-link">Choose an NPQ and provider</a>.</p>
```

Comment:
I believe this will give you the same result but lets us keep using the link helper:

```suggestion
    <p class="govuk-body"><%= govuk_link_to("Choose an NPQ and provider", root_path) %>.</p>
```